dashboard/app/models/user.rb

```diff
@@ -2081,6 +2081,7 @@ def should_see_inline_answer?(script_level)
       (script_level && UserLevel.find_by(user: self, level: script_level.level).try(:readonly_answers))
   end
 
+  # This...
```

Comment:
?

Reply:
Deleted.  Thanks

Reply:
:joy: My bad! I sometimes drop a comment like this during an investigation so I have a change marker in my code making it easy to jump back to in my editor.  I used to use [bookmarks](https://www.jetbrains.com/help/idea/navigating-through-the-source-code.html#use_bookmarks) for this, I really need to switch back to that feature!